Help/MAC_Hints.rst

```diff
@@ -1,4 +1,4 @@
-Coverage Installation Hints for OSX Users:
+Coverage Installation Hints for macOS Users:
```

Comment:
commit message says it changes OSX and MAC into macOS but in reality you are just changing OSX into macOS, so change the shortlog accordingly. also, enclose keywords such as OSX, MAC, macOS into double backticks: ``